updateList.php

```diff
@@ -0,0 +1,39 @@
+<?php
+session_start();
+
+if(isset($_GET['id'])){
+    $hobbieName = $_GET['id'];
+
+if($hobbieName === "Dancing"){$hobbieID = 1;}
+if($hobbieName === "Listening Music"){$hobbieID = 2;}
+if($hobbieName === "Cricket"){$hobbieID = 3;}
+if($hobbieName === "Singing"){$hobbieID = 4;}
+if($hobbieName === "Fooseball"){$hobbieID = 5;}
+if($hobbieName === "Reading Novels"){$hobbieID = 6;}
+
+$selfID = $_SESSION['id'];
+
+$conn = mysqli_connect("localhost", "root", "abiit@2019", "rconnect");
+
+
+$existing_row = mysqli_query($conn, "SELECT * from hobbieData where userID = $selfID AND hobbie_ID = $hobbieID");
+if(mysqli_num_rows($existing_row) == 0){
+    mysqli_query($conn, "INSERT INTO `hobbieData` (`userID`, `hobbie_ID`) VALUES ($selfID, $hobbieID)");
+    echo "canAdd";
```

Comment:
Shouldn't this be "Added"?

Reply:
Actually this is a reference/command for the JS to take action and not the user to view, that's why I named it as "canAdd"